web/packages/teleterm/src/ui/services/workspacesService/workspacesService.ts

```diff
@@ -98,12 +98,12 @@ export interface Workspace {
   // where we construct the workspace manually.
   unifiedResourcePreferences?: UnifiedResourcePreferences;
   /**
-   * Tracks whether the user has restored or discarded documents from a previous session.
+   * Tracks whether the user can restore documents from a previous session.
    * This is used to ensure that the prompt to restore a previous session is shown only once.
    *
    * This field is not persisted to disk.
    */
-  documentsRestoredOrDiscarded?: boolean;
+  canRestoreDocuments?: boolean;
```

Comment:
I feel like `hasDocumentsToReopen` might be a better name. `canRestoreDocuments` describes the ability to do something, but it's not clear whom this ability belongs to (to the user or to WorkspacesService?) and how this was decided.